pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   pedantic: any
   permission_handler: any
   photo_view: any
-  platform: any
+  platform: ^3.1.0
```

Comment:
why does any not pick the right version?

Reply:
It (`pub get`) doesn't because the checked in `pubspec.lock` file is checked in. `pub upgrade` doesn't either as the checked in lockfile is on [`2.2.1`](https://github.com/jocosocial/rainbowmonkey/blob/master/pubspec.lock#L194). `pub upgrade --major-versions` on the other hand does the upgrade and moves to `3.1.0`

Reply:
oh wow i had no idea `pub upgrade` wasn't enough.

@jason-simmons if you run `pub upgrade --major-versions` with this set to `any` does it work? if so i'd rather keep this at `any`.

Reply:
Running `pub upgrade --major-versions` upgrades the versions of various other plugins (permission_handler, flutter_local_notifications, etc.) and would require more extensive patches to the app.

Reply:
I don't see an issue with adding a specific constraint in the `pubspec.yaml` given we know there is an issue with versions lower than that.

Reply:
ok.